CHANGELOG.md

```diff
@@ -8,6 +8,15 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### BREAKING CHANGES
 
+- **CUMULUS-1672**
+  - The `cumulus` Terraform module in previous releases set a
+    `Deployment = var.prefix` tag on all resources that it managed. In this
+    release, a `tags` input variable has been added to the `cumulus` Terraform
+    module to allow resource tagging to be customized. No default tags will be
+    applied to Cumulus-managed resources. To replicate the previous behavior,
+    set `tags = { Deployment: var.prefix }` as an input variable for the
+    `cumulus` Terraform module.
```

Comment:
Can we avoid this breaking change by setting a default value that matches the previous behavior? If so, I would suggest doing so, not only to avoid a breaking change, but also to avoid having users be unaware that there is no tagging by default, and then deploy everything with _no_ tags at all.

Reply:
We talked it over in the standup and the consensus was that, as long as it was documented, this was an acceptable change.

Reply:
Should we have a corresponding PR to the template-deploy repo updating `{ Deployment: var.prefix }` to be the default value for the `tags` variable?